### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by yayner2002**
In this PR, the following tasks were implemented. I hope you will 👍  it.

- Initialise React app
- Create a directory for my components
- Add React Router and set two <Route>s and <Link>s for the app's navigation
- create Books, Categories, forms, and Book components

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './App.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
```
Comment (Ismailco) [inline src/index.js:8]: - [ ] Your `index` should have 2 `<Link>`s in the navbar which leads to the booklist and categories locations

Comment (Ismailco) [inline src/index.js:8]: - [x] Your `index` should have 2 `<Route>`s, one for the BookList component and the other for the Categories component
- [x] The BookList component should be the default page, e.g.
```
<Route exact path="/">
      <BookList />
</Route>
```
#### please implement these changes so your project respects the guidelines

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Nav />
```
Comment (Ismailco) [inline src/App.js:10]: - [x] These components should be present in your project, kindly implement them so your project meets the requirements
   - [x] `BookList` component
      - Renders a list of books using the `Book` component
      - Renders a form by using the `CreateNewBook` component
   - [x] `Book` component
      - Takes the book data as a prop and renders the title and the author
      - Has a button for removing books (the button wouldn't remove the books *yet*, this is just a placeholder for now)
  - [x] `CreateNewBook` component
      - Renders a form with inputs for the title and the author of the book
      - Has a button to submit (the button wouldn't add the book *yet*, this is just a placeholder for now)
  - [x] `Categories` component
      - should display only a button with the text "Check status"

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @yayner2002,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (yayner2002) [reply src/App.js:10]: @Ismailco I appreciate your time to review my project. i had those components that you told me to add before I did not know where they were hidden.

File: src/index.js (hunk 1 above)
Comment (yayner2002) [reply src/index.js:8]: I made these two components in the app.js module already. I don't know why it doesn't show you, brother.

**Pull request merged**